### PR TITLE
add values-zh/string.xml,that was profiles for chinese

### PR DESCRIPTION
**Pull request by binglinoioij**
add values-zh/string.xml,that was profiles for chinese

### CODE REVIEW

**Comment by hardening** [comment]
Looks good, but I don't have any Chinese knowledge.
@llyzs perhaps you could give us your opinion.

**Comment by binglinoioij** [comment]
@hardening ,Maybe it's not perfect，but i will continue to improve it.

File: client/Android/FreeRDPCore/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<resources>
+    <!-- Button labels -->
+    <string name="yes">是</string>
+    <string name="no">否</string>
+    <string name="cancel">取消</string>
+    <string name="cont">继续</string>
+    <string name="login">登陆</string>
```
Comment (realjiangms) [inline client/Android/FreeRDPCore/res/values-zh/strings.xml]: Shouldn't it be 登录? (according to your other translation for login in this file)

Comment (binglinoioij) [inline client/Android/FreeRDPCore/res/values-zh/strings.xml]: Yes, you are right. I would change it, and submit later.

Comment (binglinoioij) [inline client/Android/FreeRDPCore/res/values-zh/strings.xml]: And, i don't have any knowledge of photoshop, so i can't translate the word in the picture now. But i will try to learn some skill of photoshop to translate it. Thank you for your reply.

**Comment by hardening** [comment]
Not a big deal, let's merge that

**Pull request merged**